### DISCUSSION TITLE
Perform mathematical operations on response fields

### DISCUSSION THREAD

**Discussion by lifeisawavesorideit** [grafana/grafana-json-datasource#177] [Q&A]
Hello. First of all thank you for this excellent plug-in and your excellent work. I was wondering if there is way to perform simple mathematical operations on the response fields.

Maybe the following example makes it easier to understand what I mean by that:

My sensor values are stored in the response under `$.sensorValues[0].values[*].v`, with `$.sensorValues[0].values[*].v= [10, 15, 20, 30]`

How could I multiply, divide, add or substract these values with a certain variable so that for instance the displayed values are [100, 150, 200, 300]?

`$.sensorValues[0].values[*].v * 100` doesn't deliver the desired results.

Thanks.

**Comment by marcusolsson** [comment]
I recently added support for JSONata to be able to support this use case. It's not possible using JSONPath expressions unfortunately.
